pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -56,6 +56,9 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 	if err != nil {
 		return err
 	}
+	if infrav1.InstanceState(lb.ProvisioningStatus) != infrav1.InstanceStateActive {
```

Comment:
I thought we need check LB status instead of instance itself? 
not sure which one we want to check and use it ?

Reply:
Thank you for your comment. When I tried to reuse abstracted states I overshot the target.
Thanks for bringing clarity.